assembly/src/release/examples/stomp/python/stomppy/publisher.py

```diff
@@ -37,7 +35,7 @@
 
 for i in range(0, messages):
   conn.send(data, destination=destination, persistent='false')
-  
+
 conn.send("SHUTDOWN", destination=destination, persistent='false')
 
-conn.disconnect()
\ No newline at end of file
+conn.disconnect()
```

Comment:
I guess this is a mistake and the `disconnect()` has to happen. Can you fix please ?